.github/workflows/build.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         path: ./artifacts/TestResults/Release
 
     - name: Push NuGet packages to aspnet-contrib MyGet
-      run: dotnet nuget push "artifacts\packages\Release\Shipping\*.nupkg" --api-key ${{ secrets.MYGET_API_KEY }} --skip-duplicate --source https://www.myget.org/F/aspnet-contrib/ --symbol-source https://www.myget.org/F/aspnet-contrib/
+      run: dotnet nuget push "artifacts\packages\Release\Shipping\*.nupkg" --api-key ${{ secrets.MYGET_API_KEY }} --skip-duplicate --source https://www.myget.org/F/openiddict/api/v3/index.json
```

Comment:
Meh, that was the wrong feed 😅 
Fixed by https://github.com/aspnet-contrib/AspNet.Security.OpenId.Providers/commit/798f2f292bedc88b22061b52eac6b075f09efc09.